test/json/array.json

```diff
@@ -14,7 +14,7 @@
              "year":1937
           },
           {
-             "name":Les Demoiselles d'Avignon",
```

Comment:
all these tests are supposed to be broken.  They are a quick way to manually verify if the json linter is reporting errors/warnings correctly.